flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinatorRestoringTest.java

```diff
@@ -776,13 +775,10 @@ public void testStateRecoveryWithTopologyChange(TestScaleType scaleType) throws
                         new TestCompletedCheckpointStorageLocation());
 
         // set up the coordinator and validate the initial state
-        SharedStateRegistry sharedStateRegistry =
-                SharedStateRegistry.DEFAULT_FACTORY.create(Executors.directExecutor());
         CheckpointCoordinator coord =
                 new CheckpointCoordinatorBuilder()
                         .setExecutionGraph(newGraph)
-                        .setCompletedCheckpointStore(storeFor(
-                                        sharedStateRegistry, () -> {}, completedCheckpoint))
+                        .setCompletedCheckpointStore(storeFor(() -> {}, completedCheckpoint))
                         .setTimer(manuallyTriggeredScheduledExecutor)
```

Comment:
I don't get this change?  It looks like some github/rebasing/merge conflict issue/artefact?

Reply:
Yes, it probably came from the commit from #17179.
I don't see it after rebasing.